### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by casz**
Just fixing up some tests to reduce expensive test.

Here is our checklist for contributors. No hard requirement here, just a reminder

- [x] Please describe what you did

- [ ] Link to issue you're working on if there's a relevant one

- [ ] Did you provide a test-case to demonstrate feature actually works / issue is fixed ?

- [ ] Please also consider adding a line to [CHANGELOG.md](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/CHANGELOG.md)

### CODE REVIEW

File: plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/PrimitiveConfiguratorTest.java [hunk 1]
```diff
@@ -20,96 +22,95 @@
...
+        registry = ConfiguratorRegistry.get();
+        context = new ConfigurationContext(registry);
+    }
+
     @Test
     public void _boolean() throws Exception {
-        ConfiguratorRegistry registry = ConfiguratorRegistry.get();
-        Configurator c =registry.lookup(boolean.class);
-        final Object value = c.configure(new Scalar("true"), new ConfigurationContext(registry));
+        Configurator c =registry.lookupOrFail(boolean.class);
```
Comment (darxriggs) [inline plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/PrimitiveConfiguratorTest.java]: Spaces around `c =registry` could be corrected here and in other places.

Comment (casz) [reply plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/PrimitiveConfiguratorTest.java]: done :smiley:

**Comment by oleg-nenashev** [comment]
Just testing the webhook

**Comment by casz** [comment]
what webhook, if I may ask? 😊

**Comment by oleg-nenashev** [comment]
Gitter integration

**Comment by casz** [comment]
Sweet, I approve!

**Pull request merged**